acceptance/suites/pre_suite/foss/95_install_pdb.rb

```diff
@@ -37,7 +37,9 @@ class { 'puppetdb::master::config':
 SITEPP
 
   on(master, "chmod 644 #{sitepp}")
-  with_puppet_running_on(master, {}) do
-    on(master, puppet_agent("--test --server #{master}"), :acceptable_exit_codes => [0,2])
-  end
+  on master, puppet_apply(sitepp), :acceptable_exit_codes =>[0,2]
+  on master, "cat /etc/puppetlabs/puppet/puppet.conf"
+  # with_puppet_running_on(master, {}) do
```

Comment:
We don't want this commented code do we?

Reply:
![](https://m.popkey.co/953c40/DQb0m.gif)

Reply:
Removed